app/StyledApp.tsx

```diff
@@ -52,6 +53,13 @@ const StyledApp = () => {
     dispatch(setOsTheme());
   }, [dispatch]);
 
+  useEffect(() => {
+    eventsService
+      .syncAutoStartConfig()
+      .then(({ error }) => error && dispatch(setUiError(new Error(error))))
+      .catch((error) => dispatch(setUiError(error)));
+  }, [dispatch]);
+
```

Comment:
sync on the UI start, to track the error with permissions on different OSes and show the message to the user

Reply:
Why not just reuse the warnings approach (or move such logic in redux middleware)?